### PR TITLE
Metazoan static content update for Trialeurodes vaporariorum release 50(103)

### PR DESCRIPTION
**Pull request by ens-LCampbell**
Brand new  species ensembl core for e103 => First static content commit for this species.

### CODE REVIEW

**Review by ens-ap5 — CHANGES_REQUESTED** [review]
The Markdown files look OK, but the image name needs to be the same as the species URL, so that we can autogenerate the file paths in our code. I assume that means it should be Trialeurodes_vaporariorum_gca011764245.png

File: metazoa/species/Trialeurodes/Trialeurodes_vaporariorum/Trialeurodes_vaporariorum_gca011764245_about.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+**About *Trialeurodes vaporariorum***
+------------------------
+
+Greenhouse whitefly *Trialeurodes vaporariorum* are found predominantly in temperate regions but have a worldwide distribution. They are known to be vectors for hundreds of plant viruses [1,2], while also being resistant to a number of different insecticide groups. *T. vaporariorum* are primary insect pests for many agricultural and horticultural crops including fruits, vegetables and ornamental flowers. Adults are \~1.5 mm long but range 1-2 mm in length, white and are said to resemble tiny moths. The wings are a pale yellow, held relatively flat in relation to the leaf surface and are coated with a pure white waxy bloom giving them a 'dusted' appearance.
```
Comment (azangru) [inline metazoa/species/Trialeurodes/Trialeurodes_vaporariorum/Trialeurodes_vaporariorum_gca011764245_about.md:4]: Hi @ens-LCampbell,

Could you please delete the backslash before the tilde in the `\~1.5 mm` in the above paragraph?

Comment (ens-LCampbell) [reply metazoa/species/Trialeurodes/Trialeurodes_vaporariorum/Trialeurodes_vaporariorum_gca011764245_about.md]: No problem, that's done now. Backslash is deleted. No need to make another PR right ?

Comment (azangru) [reply metazoa/species/Trialeurodes/Trialeurodes_vaporariorum/Trialeurodes_vaporariorum_gca011764245_about.md]: 👍

**Review by azangru — APPROVED** [review]

**Pull request merged**